configuration/analyzer_config.json

```diff
@@ -2486,6 +2486,28 @@
       }
     }
   },
+  "VirusTotal_v3_Get_Behavioral_Summary": {
+    "type": "file",
+    "python_module": "vt3_sandbox.VirusTotalv3Sandbox",
+    "description": "get a file's behavioral summary from VT's integrated Sandboxes",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": true,
+    "run_hash": true,
```

Comment:
I don't think `run_hash` is necessary, am I right?

Reply:
I think the confusion arised because In the `files/{id}/behaviour_summary` endpoint as stated in VT's [docs](https://developers.virustotal.com/v3.0/reference#file-all-behaviours-summary), the `id` is the hash to identify a file.

Reply:
But this analyzer should be a file analyzer without `run_hash` because we do need the actual _file_ to be able to upload it to VT.

Reply:
If we remove the chance to upload the file, we could just move this analyzer to an `observable` analyzer and add the related `file` analyzer with `run_hash` option. This would be more clear because otherwise we do not have support for hash analysis for this analyzer.

About this, I am also making some changes to Intezer and MWDB analyzers because they have the same problem. They can be used only as file analyzer and not as observable (hash only) analyzers

Reply:
@Eshaan7 I thought the `id` was the hash of the file and that this endpoint just `GET`s the summary from an already existing file in the database. Else, it's a `404`, in which case we'll have to upload it first.